website/source/upgrade-guides/0-7.html.markdown

```diff
@@ -0,0 +1,135 @@
+---
+layout: "downloads"
+page_title: "Upgrading to Terraform 0.7"
+sidebar_current: "upgrade-guides-0-7"
+description: |-
+  Upgrading to Terraform v0.7
+---
+
+# Upgrading to Terraform v0.7
+
+Terraform v0.7 is a major release, and thus includes some backwards incompatibilities that you'll need to consider when upgrading. This guide is meant to help with that process.
+
+The goal of this guide is to cover the most common upgrade concerns and issues that would benefit from more explanation and background. The exhaustive list of changes will always be the [Terraform Changelog](https://github.com/hashicorp/terraform/blob/master/CHANGELOG.md). After reviewing this guide, review the Changelog to check on specific notes about the resources and providers you use.
+
+## Plugin Binaries
+
+Before v0.7, Terraform's built-in plugins for providers and provisioners were each distributed as separate binaries.
+
+```
+terraform               # core binary
+terraform-provider-*    # provider plugins
+terraform-provisioner-* # provisioner plugins
+```
+
+These binaries needed to all be extracted to somewhere on your `$PATH` for Terraform to work.
```

Comment:
"In order for Terraform to work, each of these binaries must have been available on your `$PATH`, or in the `~/.terraform.d` directory."?